contributors/guide/style-guide.md

```diff
@@ -0,0 +1,683 @@
+---
+title: Documentation Style Guide
+---
+
+# Documentation Style Guide
+
+This style guide is for content in the Kubernetes github [community repository].
+It is an extension of the [Kubernetes documentation style-guide].
+
+These are **guidelines**, not rules. Use your best judgement.
+
+- [Cheatsheet](#cheatsheet)
+- [Content design, formatting, and language](#content-formatting-and-language)
+  - [Contact information](#contact-information)
+  - [Dates and times](#dates-and-times)
+  - [Diagrams, images and other assets](#diagrams-images-and-other-assets)
+  - [Document Layout](#document-layout)
+  - [Formatting text](#formatting-text)
+  - [Language, grammar, and tone](#language-grammar-and-tone)
+  - [Moving a document](#moving-a-document)
+  - [Punctuation](#punctuation)
+  - [Quotation](#quotation)
+- [Markdown formatting](#markdown-and-formatting)
+  - [Code Blocks](code-blocks)
+  - [Emphasis](#emphasis)
+  - [Headings](#headings)
+  - [Horizontal Lines](#horizontal-lines)
+  - [Line Length](#line-length)
+  - [Links](#links)
+  - [Lists](#lists)
+  - [Metadata](#metadata)
+  - [Tables](#tables)
+- [Attribution](#attribution)
+
+
+## Cheatsheet
+
+### Cheatsheet: Content design, formatting, and language
+
+**[Contact information:](#contact-information)**
+- Use official Kubernetes contact information.
+
+**[Dates and times:](#dates-and-times)**
+- Format dates as `month day, year`. (December 13, 2018)
+- When conveying a date in numerical form, use [ISO 8601] Format: `yyyy-mm-dd`.
+- Use the 24 hour clock when referencing time.
+- Times for single events (example: KubeCon) should be expressed in an absolute
+  timezone such as Pacific Standard Time (PST) or Coordinated Universal Time
+  (UTC).
+- Times for reoccurring events should be expressed in the PT (Pacific Time)
```

Comment:
- If it is the intent that a recurring event will follow Daylight Savings Time (DST), then the time should be expressed with a timezone that follows DST. This way, recurring events will not need to be updated twice a year.

Reply:
Updated to read: "Times for reoccurring events should be expressed in a time zone that follows Daylight Savings Time (DST) such as Pacific Time (PT) or Eastern Time (ET)."

The explanation regarding why (not having to update the doc) is expanded upon in the time formatting section.

Although,  I am unsure of how worthwhile it is to have the "cheatsheet" portion at the top and the full thing further down as the explanations and examples add more context.